src/patterns/MenuBar.vue

```diff
@@ -83,6 +83,22 @@ export default {
         this.$emit("input", val)
       },
     },
+    parsedMenuItems() {
```

Comment:
I think it would be easier to read that there are child elements if it was slightly indented. We could add a class and give it some extra padding on the left.